onnxruntime/contrib_ops/cpu/quantization/dynamic_quantize_lstm.cc

```diff
@@ -33,7 +28,6 @@ class DynamicQuantizeLSTM : public OpKernel, public LSTMBase {
   bool is_R_signed_;
 };
 
-#ifdef MLAS_SUPPORTS_PACKED_GEMM_U8X8
 Status DynamicQuantizeLSTM::TryPackWeights(const Tensor& weights, PackedWeights& packed_weights, bool& is_packed, bool& is_weight_signed) {
```

Comment:
Could you also clean up this [https://github.com/microsoft/onnxruntime/blob/87ec1f6208f2bb90b079baf444ad942aecfb56ee/onnxruntime/core/providers/cpu/rnn/rnn_helpers.cc#L289](https://github.com/microsoft/onnxruntime/blob/87ec1f6208f2bb90b079baf444ad942aecfb56ee/onnxruntime/core/providers/cpu/rnn/rnn_helpers.cc#L289)?

Reply:
Are you looking at an older branch here? The file you mention was cleaned up (it wouldn't be able to build with the old function signature).

Reply:
It was in the file list later. 

---
In reply to: [590637096](https://github.com/microsoft/onnxruntime/pull/6916#discussion_r590637096) [](ancestors = 590637096)